ncbi_genome_download/__main__.py

```diff
@@ -2,9 +2,9 @@
 import argparse
 import logging
 
-from . import __version__
-from . import download
-from . import EDefaults as dflt
+from ncbi_genome_download import __version__
+from ncbi_genome_download import download
+from ncbi_genome_download import EDefaults as dflt
```

Comment:
I encountered locally the issue I fixed. Maybe it just affects Python 3.
Working on the other issues.